pkg/kritis/secrets/secrets_test.go

```diff
@@ -31,56 +31,99 @@ import (
 var pub, priv = createKeys("good")
 var pgpKey, err = NewPgpKey(priv, "", pub)
 
+var pub2, priv2 = passphraseProtectedSecretPublicKey, passphraseProtectedSecretPrivateKey
+var goodPassphrase, badPassphrase = passphraseProtectedSecretPassphrase, "bad-passphrase"
+var pgpKey2, err2 = NewPgpKey(priv2, goodPassphrase, pub2)
+
 var tests = []struct {
 	name       string
 	secretName string
 	shdErr     bool
 	expected   *PGPSigningSecret
 }{
-	{"good", "good-sec", false, &PGPSigningSecret{SecretName: "good-sec", PgpKey: pgpKey}},
-	{"bad1", "bad1-sec", true, nil},
-	{"bad2", "bad2-sec", true, nil},
-	{"notfound", "not-present", true, nil},
+	{"good-nopass", "good-sec-nopass", false, &PGPSigningSecret{SecretName: "good-sec-nopass", PgpKey: pgpKey}},
```

Comment:
consider removing the `secretName` field and just re-using the `name` in the tests, since they're so similar

Reply:
Good point. Ack.